hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -49,10 +49,12 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import javax.management.ObjectName;
+import org.apache.hadoop.conf.StorageUnit;
```

Comment:
Only if the patch is updated for any other reason: please use `org.apache.hadoop.hdds.conf.StorageUnit`.